forest/src/cli/config.rs

```diff
@@ -34,9 +34,15 @@ pub struct Config {
 
 impl Default for Config {
     fn default() -> Self {
+        let dir = ProjectDirs::from("com", "ChainSafe", "Forest").expect("project directories couldn't be found and that FOREST_CONFIG_PATH must be set manually.");
```

Comment:
`failed to find project directories, please set FOREST_CONFIG_PATH environment variable manually.`